src/assets/simp-adapter/build/simp-adapter.spec

```diff
@@ -36,8 +36,6 @@ Summary: SIMP Adapter for the Puppet Enterprise Puppet Installation
 License: Apache-2.0
 Requires: rsync
 Requires(post): puppet-agent
-Requires(post): pe-puppetserver
```

Comment:
One of those days....Requires(post) are in the simp-adapter-foss RPM, but race condition is still a problem.  Regardless, would be more clear if there was symmetry in the Requires.